catalog/java/io/material/catalog/card/SelectableCardsAdapter.java

```diff
@@ -99,13 +99,6 @@ private void bind(Item item, int position) {
       if (selectionTracker != null) {
         bindSelectedState();
       }
-      // Set an OnLongClickListener for accessibility
-      materialCardView.setOnLongClickListener(
-          v -> {
-            selectionTracker.setItemsSelected(
```

Comment:
I think the right fix should be adding an if check instead:

if (materialCardView.isChecked()) {
  selectionTracker.xxxxxx....
  return true;
}
return false;

Reply:
`SelectionTracker` manages the list item selections on its own, it shouldn't need additional `onClickListener`.